uint/benches/bigint.rs

```diff
@@ -88,6 +88,42 @@ fn u512_mul_small(b: &mut Bencher) {
     });
 }
 
+#[bench]
+fn u512_rem(b: &mut Bencher) {
+    b.iter(|| {
+        let one = U512([8326634216714383706, 15837136097609390493, 13004317189126203332, 7031796866963419685, 12767554894655550452, 16333049135534778834, 140317443000293558, 598963]);
```

Comment:
You should move the `let one/two` definitions outside of the `b.iter(|| { .. })` loop. Otherwise you also benchmark the construction of those. Please do this for all the new benchmark functions.